tests/unittest_observer/FaceliftObserverTest.cpp

```diff
@@ -0,0 +1,76 @@
+#include <gtest/gtest.h>
+#include "IPCProxyBase.h"
+#include "InterfaceBase.h"
+#include <QSignalSpy>
+
+namespace {
+
+using namespace facelift;
+
+class Counter
+{
+public:
+    Counter() { m_value = 0; }
+
+    void incValue() {
+        m_value++;
```

Comment:
why postfix increment ?

Reply:
In this case it is no matter, result will be a same. But I agree the postfix is more expensive, therefore necessary to change increment.

Reply:
I'm not sure (and do not argue) that it doesn't matter, but anyway - it's better with the prefix form